### PR TITLE
Unify tooltips

### PR DESCRIPTION
**Pull request by rubo**
- Made tooltips the same across the app
- Moved treemap color to `tailwind.config.js`
- Revised EIGEN formatting

### CODE REVIEW

File: src/lrt/LRTDistribution.jsx [hunk 1]
```diff
@@ -404,7 +404,7 @@ export default function LRTDistribution({ data, height }) {
                 </li>
               ))}
           </ul>
-          <div className="mt-2 flex flex-row px-2 text-sm">
+          <div className="mt-2 flex flex-row px-2">
```
Comment (vincenthongzy) [inline src/lrt/LRTDistribution.jsx:407]: should we show total in the homepage stacked area graph too?

**Review by vincenthongzy — APPROVED** [review]

**Review by Zishan-7 — APPROVED** [review]

File: src/lrt/LRTDistribution.jsx (hunk 1 above)
Comment (rubo) [reply src/lrt/LRTDistribution.jsx:407]: Good question... Let me check.

File: src/home/charts/EigenLayerTVLOvertimeChart.jsx [hunk 2]
```diff
@@ -58,19 +59,18 @@ export default function EigenLayerTVLOvertimeChart({ eigenLayerTVL, height }) {
           0,
           Math.max(
             ...eigenLayerTVL.map(d => {
-              let value = 0;
+              const value = Number(
+                formatEther(BigInt(d.ethTVL) + BigInt(d.lstTVL))
+              );
 
-              value += parseFloat(BigInt(d.ethTVL) / BigInt(1e18));
-              value += parseFloat(BigInt(d.lstTVL) / BigInt(1e18));
```
Comment (rubo) [inline src/home/charts/EigenLayerTVLOvertimeChart.jsx:64]: @Zishan-7 FYI

File: src/lrt/LRTDistribution.jsx (hunk 1 above)
Comment (rubo) [reply src/lrt/LRTDistribution.jsx:404]: Added

**Pull request merged**